superset-frontend/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -539,7 +539,15 @@ class SqlEditor extends React.PureComponent {
   }
 
   renderQueryLimit() {
-    const menuDropdown = (
+    // Update the limit dropdown with SQL_ROW_MAX value
+    const { maxRow } = this.props;
```

Comment:
to be safe, should we set a default if not defined?

Reply:
This should be `DISPLAY_MAX_ROW`, not `SQL_ROW_MAX`. `SQL_MAX_ROW` is for results backend and CSV export.

Also, what happens if the user sets a `DEFAULT_SQLLAB_LIMIT` that's not a multiple of 10?

Reply:
It looks like this limit control is getting applied to the UI results and to CSV export. Our main concern is with CSV export so that's why I mentioned the `SQL_ROW_MAX`. If I run a query with a limit 10 in the UI dropdown (with no limit in the query) and run it then download csv I get only 10 rows in the CSV. It sounds like this is the same as the old functionality for the limit UI control to affect the csv and the UI results. I wasn't aware that changed, but if there's some other setting we need to make the UI limit only apply to the ui results let me know.